wpilib-sys/src/hal_call.rs

```diff
@@ -124,13 +118,14 @@ impl<T> HalMaybe<T> {
 /// the `Err` variant.
 #[macro_export]
 macro_rules! hal_call {
-    ($function:ident($($arg:expr),*)) => {{
+    ($function:ident($($arg:expr,)*)) => {{
```

Comment:
I'm not quite sure how this is different from the other pattern.
https://doc.rust-lang.org/reference/macros-by-example.html shows patterns declared like the old version.
Right now my guess is that this case requires a trailing comma, and the other requires no trailing comma. If that's the case, adding a comment might be helpful.

Reply:
If this were using Rust 2018, we could consolidate the two patterns into one (without allowing an arbitrary number of trailing commas). Alas, this is not 2018 edition.

Reply:
I can't believe I forgot to update to 2018 edition! I'll start work on that, and then with a rebase we could remove the extra pattern. Unless there's a reason to stay on 2015?

Reply:
Sounds good.

Reply:
Done!